README.md

```diff
@@ -1,69 +1,89 @@
 # usb-filesystem
-example of using a brightsign device as a filesystem over USB
+Example using BrightSign software to host a USB filesystem in Javascript to connected peripherals.
 
-1. Overview
+## Overview
 
-BrightSign players support USB role reversal modes through the USB-C ports on BrightSign players. The BrightSign would be a USB client for an external device to utilize. The BrightSign can dynamically build a filesystem that is needed for the USB client to reference as the USB image. If needed, please ask BrightSign for an example presentation that demonstrates how to utilize the commands outlined below. 
+BrightSign players support USB role reversal modes through the USB-C ports on BrightSign players except for LS models. The BrightSign would be a USB client for an external device to utilize. The BrightSign can dynamically build a filesystem that is needed for the USB client to reference as the USB image.
```

Comment:
The below system requirements says XD and XT series4. If only intended to work on those two models, we can say XD and XT models to match here. Otherwise, LS may not be the only exception(e.g. AU).

Reply:
Updated